src/Client.php

```diff
@@ -384,9 +448,17 @@ public function sendMessage(Message $message)
    * @param \Maknz\Slack\Message $message
    * @return void
    */
-  public function queueMessage(Message $message)
+  public function queueMessage(Message $message, $numRetries = self::MAX_RETRY_ATTEMPTS)
```

Comment:
Its a public function and anyone can use the client class. Nevertheless, will remove it here, since it becomes the responsibility of the caller to pass in the right values